### ISSUE TITLE
Opt-in for Tinder Groups

### ISSUE THREAD

**Issue by gwendall** [tinderjs/tinderjs#38]
This feature doesn't seem to be available in all countries. However I was wondering: isn't there an API call available for anyone to turn it on yet ?

**Comment by mrmayfield** [comment]
Haven't used that feature, haven't been on Tinder for a while. Maybe you want to sniff it out using Charles Proxy?

**Comment by gwendall** [comment]
I am not in one of the countries where it's available right now (US and Australia) and it can only be done via the app. I could try to spoof my location with `updateLocation` to get there and inspect the app through a proxy with the geolocation turned off on my phone but I think the app gets blocked when location is turned-off. Will keep you posted.

**Comment by mrmayfield** [comment]
Yup sounds good. I'm in Canada so I wouldn't be able to sniff it out either. Hopefully someone here is US or Australia based.

**Comment by gwendall** [comment]
There you go. It is done by doing a POST to `api.gotinder.com/profile` with `{ squads_discoverable: Bool }`

**Comment by mrmayfield** [comment]
Awesome. Want to submit a PR for this guy?

**Comment by gwendall** [comment]
Here it is https://github.com/tinderjs/tinderjs/pull/39
